packs/feat-effects/effect-expel-maelstrom-failure.json

```
@@ -0,0 +1,68 @@
+{
+    "_id": "lc8ez8O4eN73hNpR",
+    "img": "icons/magic/control/energy-stream-link-spiral-white.webp",
+    "name": "Effect: Expel Maelstrom (Failure)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.actionspf2e.Item.Expel Maelstrom]</p>\n<p>The creature takes a -2 status penalty to all saving throws and skill checks for 10 minute.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.actionspf2e.Item.Expel Maelstrom]</p>\n<p>The creature takes a -2 status penalty to all saving throws and skill checks for 10 minutes.</p>"
```
Typo here, will fix tomorrow. Anyone feel free to fix it if it's otherwise good to go though.